frontend/src/metabase/visualizations/echarts/pie/option/index.ts

```diff
@@ -0,0 +1,29 @@
+import type { EChartsOption } from "echarts";
+
+import type {
+  ComputedVisualizationSettings,
+  RenderingContext,
+} from "metabase/visualizations/types";
+
+import type { PieChartModel } from "../model/types";
+import { SUNBURST_SERIES_OPTIONS } from "./constants";
+
+export function getPieChartOption(
+  chartModel: PieChartModel,
+  settings: ComputedVisualizationSettings,
+  renderingContext: RenderingContext,
+): EChartsOption {
+  return {
+    textStyle: {
+      fontFamily: renderingContext.fontFamily,
+    },
+    series: {
+      ...SUNBURST_SERIES_OPTIONS,
+      data: chartModel.slices.map(s => ({
+        value: s.value,
+        name: s.key,
```

Comment:
I think it should be a slice percentage here when the related settings enabled. For now we don't show dimension values in slices

Reply:
Yup I fix this in https://github.com/metabase/metabase/pull/34869